roles/mistral/tasks/main.yml

```diff
@@ -1,18 +1,19 @@
 ---
-
 - name: Install latest st2mistral
   become: yes
-  apt: 
+  package:
     name: st2mistral
     state: latest
```

Comment:
Yeah, it's good, considering condition in this block.
The idea here to install `latest` mistral eg. get upgrades.